core/src/test/scala/com/salesforce/op/ModelInsightsTest.scala

```diff
@@ -406,6 +407,21 @@ class ModelInsightsTest extends FlatSpec with PassengerSparkFixtureTest {
         }
         insights.trainingParams.toJson() shouldEqual deser.trainingParams.toJson()
         insights.stageInfo.keys shouldEqual deser.stageInfo.keys
+
+        // check that config is correctly serialized and deserialized
+
+        def getRawFeatureFilterConfig(modelInsights: ModelInsights): Map[String, Any] = {
+          modelInsights.stageInfo("rawFeatureFilter") match {
+            case configInfo: Map[String, Map[String, Any]] => configInfo.getOrElse("params", Map.empty[String, Any])
+            case _ => Map.empty[String, Any]
+          }
+        }
+
+        (getRawFeatureFilterConfig(insights), getRawFeatureFilterConfig(deser)) match {
+          case (paramsMapI, paramsMapD) =>
+            paramsMapI.keys shouldEqual paramsMapD.keys
+            paramsMapI.keys.foreach { `key` => paramsMapI(key) shouldEqual paramsMapD(key) }
```

Comment:
@leahmcguire @Jauntbox 

Trying to serialize / deserialize RFF config with json4s as part of `stageInfo` in `ModelInsights`

But currently running into issue with trying to serialize `Double.PositiveInfinity`.

At this line, I'm getting error:

> `org.scalatest.exceptions.TestFailedException: Infinity did not equal "Infinity"`

Any thoughts?

Reply:
If you look at the way the stage parameters are saved they should be converted to strings
https://github.com/salesforce/TransmogrifAI/blob/master/core/src/main/scala/com/salesforce/op/ModelInsights.scala#L679

Reply:
`SpecialDoubleSerializer` is your friend

Reply:
I've updated to now convert to strings